src/blog/2024/01/speech-driven-chatbot-with-node-red.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Speech-Driven Chatbot System with Node-RED
+subtitle: Guide to building speech-driven chatbot using Node-RED, speech recognition, and Dashboard 2.0.
+description: Guide to building speech-driven chatbot using Node-RED, speech recognition, and Dashboard 2.0.
+date: 
```

Comment:
@Hasmin-AC Can you decide when we publish this?

Reply:
Tuesday, Jan 30th would be good.

Reply:
```suggestion
date: 30-01-20024
```